tests/conftest.py

```diff
@@ -28,20 +55,25 @@ def _create_mgf(peptides, mgf_file, random_state=42):
         The MGF file to create.
     random_state : int or numpy.random.Generator, optional
         The random seed. The charge states are chosen to be 2 or 3 randomly.
+    c_mod : bool, optional
```

Comment:
**todo:** This can be generalized. Carbamidomethylation is not necessarily the only relevant modification.

Reply:
Generalized to accept a dictionary of modifications